great_expectations/rule_based_profiler/helpers/util.py

```diff
@@ -310,6 +310,7 @@ def get_resolved_metrics_by_key(
         Dictionary of the form {
             "my_key": List[MetricConfiguration],  # examples of "my_key" are: "my_column_name", "my_batch_id", etc.
         }
+        force_no_progress_bar: (bool) if True, prevent all "Calculating Metrics" output; (False by default).
```

Comment:
nit: I'd prefer this to not be a double negative. and be something like `use_progress_bar`.

Reply:
@billdirks Feel free to change (and discuss with @kenwade4).  I have not thought of a use-case for this flag as of yet.  Thanks.